### ISSUE TITLE
Ensure Links Are Distinguishable Without Relying Solely on Colour for Accessibility

### ISSUE THREAD

**Issue by ipreencekmr** [expressjs/expressjs.com#1792]
When a link is embedded within surrounding text, it should be easily distinguishable without relying solely on colour. Users with colour blindness or low vision may struggle to perceive colour differences, making it difficult or impossible to identify the link. To improve accessibility, it is recommended to underline links, ensuring they stand out clearly from the surrounding text.

<img width="275" alt="Image" src="https://github.com/user-attachments/assets/ada303da-2973-44d7-98a4-16bac6d326f9" />

<img width="858" alt="Image" src="https://github.com/user-attachments/assets/4325ff0c-6896-4234-847d-b1ab576934f4" />

<img width="931" alt="Image" src="https://github.com/user-attachments/assets/1af6e8b5-f41a-4346-a513-47c019f4ee05" />

<img width="1138" alt="Image" src="https://github.com/user-attachments/assets/0ad83539-a6cd-47c0-af7d-654a3be3f836" />

**Comment by ipreencekmr** [comment]
[PR](https://github.com/expressjs/expressjs.com/pull/1796) raised for fix

**Comment by bjohansebas** [comment]
fixed #1796
